pages/docs/react/latest/context.mdx

```diff
@@ -104,26 +101,21 @@ Using context, we can avoid passing props through intermediate elements:
 // src/App.res
 
 module ThemeContext = {
-  type theme = Light | Dark;
+  type theme = Light | Dark
   let context = React.createContext(Light)
 
   module Provider = {
-    let provider = React.Context.provider(context)
-
-    @react.component
-    let make = (~value, ~children) => {
-      React.createElement(provider, {"value": value, "children": children})
-    }
+    let make = React.Context.provider(context)
```

Comment:
`type props` is missing

Reply:
I think we don't need to add the `type props` for context. It seems inferred properly with https://github.com/rescript-lang/rescript-react/blob/2c5dea342645921b3efe8b47022fe1d73012ad6e/src/React.res#L82

Reply:
Ah, great, didn't know that, I thought one always had to define a type `props` in the module like one always had to define a function `makeProps` in V3. 👍